webpack.config.ts

```diff
@@ -372,9 +372,9 @@ const appConfig: Configuration = {
               configOverwrite: {
                 compilerOptions: {incremental: true},
```

Comment:
fun fact, incremental doesn't actually work because its not setup correctly. i think it needs a path for where to put that file or something